strr-examiner-web/app/components/Host/EditRentalUnitForm.vue

```diff
@@ -0,0 +1,267 @@
+<script setup lang="ts">
+
+const exStore = useExaminerStore()
+const {
+  rentalUnitAddressToEdit,
+  hasUnsavedRentalUnitChanges,
+  rentalUnitAddressSchema
+} = storeToRefs(exStore)
+const strrModal = useStrrModals()
+const { t } = useI18n()
+const confirmUnchangedModal = ref<ConfirmModal | null>(null)
+
+const route = useRoute()
+const isApplicationRoute = computed(() => route.path.includes('/examine/'))
+const isRegistrationRoute = computed(() => route.path.includes('/registration/'))
+const { isFeatureEnabled } = useFeatureFlags()
+const canEditApplicationAddress = isFeatureEnabled('enable-examiner-edit-address-application')
```

Comment:
Do we allow edit address for applications?

Reply:
Feature flag controls it and it will not allow it in any envs other than dev.